swarm/network/stream/delivery.go

```diff
@@ -124,11 +125,11 @@ func (s *SwarmChunkServer) Close() {
 
 // GetData retrives chunk data from db store
 func (s *SwarmChunkServer) GetData(ctx context.Context, key []byte) ([]byte, error) {
-	chunk, err := s.chunkStore.Get(ctx, storage.Address(key))
+	ch, err := s.chunkStore.Get(ctx, chunk.ModeGetRequest, storage.Address(key))
```

Comment:
this is supposed to be ModeSync no?
retrive -> retrieve

Reply:
ModeGetSync is used in SwarmSyncerServer.GetData in handleWantedHashesMsg. Should SwarmChunkServer be used for requests only?